src/backend/codegen/include/codegen/utils/utility.h

```diff
@@ -32,7 +33,7 @@ namespace gpcodegen {
  * @return A pointer to the specified argument, or NULL if the specified
  *         position was beyond the end of function's arguments.
  **/
-llvm::Argument* ArgumentByPosition(llvm::Function* function,
+static llvm::Argument* ArgumentByPosition(llvm::Function* function,
```

Comment:
It's in header file. So if it is not static, then it gives error as "multiple definition of symbol" as this header file is included from different place.